### PR TITLE
Create vwrs-metrics.md

### PR DESCRIPTION
**Pull request by diana-macrometa**
Creating a page for VWRS metrics.

### CODE REVIEW

File: docs/photoniq/vwrs/vwrs-metrics.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+You can retrieve metrics using the following endpoints:
+
+- Get Metrics for a Domain - Retrieve the metrics for a specific domain. Set start and end times to get metrics for a specific time period.
+- Get All Metrics - Retrieve metrics for all domains. Set start and end times to get metrics for a specific time period.
+
+## Available Metrics
+
+The metrics response body includes the following properties:
+
+- **avg_waiting_time**: (integer) The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/vwrs-metrics.md]: I would remove the (type) after each description.  This is a very explicit thing that probably is not needed here.

File: docs/photoniq/vwrs/vwrs-metrics.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  {
+    "_key": "www.example1.com:2023-08-21_07:50:00",
+    "avg_waiting_time": 0,
+    "domain_key": "www.example1.com",
+    "duplicate_request_rate": 100,
+    "peak_queue_length": 100,
+    "queue_abandonment_rate": 0,
+    "request_success_rate": 100,
+     "usage_requests": 6000,
+    "timestamp": 1692604200000
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/vwrs-metrics.md]: Strange.  The timestamp is in milliseconds.  I assume you copied this from somewhere.  We should fix this there too.
Remove the last three zeros from the timestamp to get it in seconds (as defined above).
You can verify the timestamp here: https://www.epochconverter.com/

All three samples have the timestamp in milliseconds.  Remove the last three zeros to get it to seconds for all of these.
@rahul-naik1 does our swagger file contain these examples?  If so, we should probably fix the timestamp.

**Review by joubert-macrometa — CHANGES_REQUESTED** [review]

File: docs/photoniq/vwrs/vwrs-metrics.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/photoniq/vwrs/vwrs-metrics.md]: I copied this example from the Box file.

File: docs/photoniq/vwrs/vwrs-metrics.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 100
+title: Get VWRs Metrics
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) has built-in metrics that allow you to retrieve waiting room telemetry per waiting room, per minute. Data is stored for the past 90 days.
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/vwrs-metrics.md:6]: This got me thinking:  VWR vs VWRS.
What is the S?  "Get the VWRs metrics" ==> "Get the Virtual Waiting Room metrics"  What happened to the "s"

I think originally the S was Service ==> Virtual Waiting Room Service (is that right @rahul-naik1)

Should this then start with "Macrometa PhotonIQ Virtual Waiting Room Service..."
and then what about  "Rooms" 

@diana-macrometa you know, me overthinking probably, but thought I would just say it out loud.  You do with it what you think is best.

Comment (diana-macrometa) [reply docs/photoniq/vwrs/vwrs-metrics.md:6]: The "s" just makes it plural.

Comment (diana-macrometa) [reply docs/photoniq/vwrs/vwrs-metrics.md:6]: @nicolewilliams88 and @macrochrissie , any comment?

**Review by joubert-macrometa — APPROVED** [review]

File: docs/photoniq/vwrs/vwrs-metrics.md (hunk 2 above)
Comment (rahul-naik1) [reply docs/photoniq/vwrs/vwrs-metrics.md]: Let me check what we returning

Comment (rahul-naik1) [reply docs/photoniq/vwrs/vwrs-metrics.md]: @joubert-macrometa
We are storing values in milliseconds

File: docs/photoniq/vwrs/vwrs-metrics.md (hunk 3 above)
Comment (rahul-naik1) [reply docs/photoniq/vwrs/vwrs-metrics.md:6]: @joubert-macrometa Yes, it is for Service

**Pull request merged**